### PR TITLE
Add WebSync.app v2.8.2

### PR DESCRIPTION
**Pull request by stevenmaguire**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
The should be some separation of elements, in the manner of

```ruby
cask :v1 => 'polar-websync' do
  version '2.8.2'
  sha256 :no_check

  url "https://www.polarpersonaltrainer.com/downloads/websync_#{version}.dmg"
  name 'Polar WebSync Software'
  homepage 'http://www.polar.com/us-en/support/downloads/Polar_WebSync_Software'
  license :closed

  pkg 'Polar WebSync.pkg'

  zap :delete => [
                  '/Applications/WebSync.app'
                 ]
end
```

In addition (and more importantly, it’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that. Do you need any help?

**Comment by stevenmaguire** [comment]
@vitorgalvao Thanks for catching that. This is my first foray into Cask creation. I have pushed another version that includes, what I believe to be the full routine. Please review. Let me know if there is anything else that I need to address. This is fun!

File: Casks/polar-websync.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'polar-websync' do
+  version '2.8.2'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/polar-websync.rb]: `sha256` also needs to be updated. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/polar-websync.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  url "https://www.polarpersonaltrainer.com/downloads/websync_#{version}.dmg"
+  name 'Polar WebSync Software'
+  homepage 'http://www.polar.com/us-en/support/downloads/Polar_WebSync_Software'
+  license :closed
+  caveats "Installation of this application requires user input."
```
Comment (vitorgalvao) [inline Casks/polar-websync.rb]: What input does it require? If it hangs the installation, we”ll need to take care of it.

File: Casks/polar-websync.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+  pkg 'Polar WebSync.pkg'
+
+  uninstall :pkgutil => 'fi.polar.websync.pkg',
+            :delete => '/Applications/WebSync.app',
```
Comment (vitorgalvao) [inline Casks/polar-websync.rb]: Doesn’t this get removed by `pkugutil`?

File: Casks/polar-websync.rb (hunk 1 above)
Comment (stevenmaguire) [inline Casks/polar-websync.rb]: Ah good tip. I'll take care of that.

File: Casks/polar-websync.rb (hunk 2 above)
Comment (stevenmaguire) [inline Casks/polar-websync.rb]: The input doesn't hang, it just requires that you select some configuration items before the application is usable.

File: Casks/polar-websync.rb (hunk 3 above)
Comment (stevenmaguire) [inline Casks/polar-websync.rb]: Perhaps. I will try. I saw another example that was using this pattern.

Comment (stevenmaguire) [inline Casks/polar-websync.rb]: A test here shows that the removing that key does not remove the WebSync.app during uninstall

**Comment by vitorgalvao** [comment]
The `caveat` should be removed. It’ll be shown too late to be relevant, anyway. I’ve tried the install, and see what you mean. I *does* hang the installation, and with a GUI, even, which is a big no-no.

We need some other way to handle this one. Pinging @phinze @ndr-qef @rolandwalker @radeksimko.

**Comment by stevenmaguire** [comment]
Caveat now removed. This one definitely appears to be sticky due to the user action required. I see that you've signaled for reinforcements, and I am curious what some of the potential solutions might involve from a high level? I am just now becoming familiar with the toolset included in the project and this seems like a great opportunity to find out what else this puppy can do!

**Comment by vitorgalvao** [comment]
Well, we don’t have a solution for these issues, and therein lies the problem. We need to discuss something that ideally could be used in the future.

**Comment by radeksimko** [comment]
Just so we're all on the same page in this thread:

screen shot 2015-05-09 at 10 16 43
is displayed while brewcask is waiting for user:

screen shot 2015-05-09 at 10 16 59

I don't think we can choose any one of these two options on behalf of the user.
The good message here is, that it seems to be a standard `pkg` format, therefore I think https://github.com/caskroom/homebrew-cask/pull/10137 would help here.

Until we have a solution for these cases, I'd label this PR and **not merge it yet**. The reason is that it would cause disruption for any user expecting fully automated installation (which at the end is the ultimate goal of brewcask).

cc @retrography

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 811180bb920931253b1f192992a810ef7f7a96ae. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by stevenmaguire** [comment]
Awesome. Thank you! I will dig in a it and see how things are working now.

**Comment by vitorgalvao** [comment]
Well, it just means you’ll have to run the installer manually.

**Comment by stevenmaguire** [comment]
Oh! Well, ok.

**Comment by vitorgalvao** [comment]
Best we can do for now, unfortunately, and it’s unlikely we’ll be able to do better any time soon.

**Comment by stevenmaguire** [comment]
cheer up

**Comment by vitorgalvao** [comment]
Thank you for that. Made me laugh.

**Pull request closed without merging**